chainerx/_docs/routines.py

```diff
@@ -1342,6 +1342,25 @@ def _docs_math():
     output array to the input array ``x``.
 
 .. seealso:: :data:`numpy.arctan`
+""")
+
+    _docs.set_doc(
+        chainerx.arctan2,
+        """arctan2(x1, x2)
+Element-wise arc tangent of x1/x2 choosing the quadrant correctly.
+
+Args:
+    x1 (~chainerx.ndarray): Input array.
+    x2 (~chainerx.ndarray): Input array.
+
+Returns:
+    :class:`~chainerx.ndarray`: Returned array: :math:`y = \\arctan (x1/x2)`.
```

Comment:
This expression seems incorrect. How about explaining in sentences without using mathematical expressions?

Reply:
Have replaced it with `Element-wise arc tangent of x1/x2`.